packages/grid/_modules_/grid/components/columnHeaders/ColumnHeaderFilterIcon.tsx

```diff
@@ -43,13 +43,21 @@ export function ColumnHeaderFilterIcon(props: ColumnHeaderFilterIconProps) {
   }
 
   const iconButton = (
-    <IconButton onClick={toggleFilter} color="default" aria-label="Show Filters" size="small">
+    <IconButton
+      onClick={toggleFilter}
+      color="default"
+      aria-label={apiRef!.current.getLocaleText('columnHeaderFiltersLabel')}
+      size="small"
+    >
       {filteredColumnIconElement}
     </IconButton>
   );
 
   return (
-    <Tooltip title={`${counter} active filter(s)`} enterDelay={1000}>
+    <Tooltip
+      title={`${counter} ${apiRef!.current.getLocaleText('columnHeaderFiltersTooltipActive')}`}
```

Comment:
The position of the number relative to the label can vary between locales.

In English, sure it's "1 filter", but in other, it's "filter 1", so:

```suggestion
      title={apiRef!.current.getLocaleText('columnHeaderFiltersTooltipActive')(counter)}
```